src/BloomExe/Collection/BookCollection.cs

```diff
@@ -189,7 +187,34 @@ protected Color CoverColor
 			set { throw new NotImplementedException(); }
 		}
 
-		public static string DownloadedBooksCollectionNameInEnglish = "Books From BloomLibrary.org";
+		public const string DownloadedBooksCollectionNameInEnglish = "Books From BloomLibrary.org";
+		private const string InstalledTemplatesCollectionNameInEnglish = "Templates";
+
+		public bool IsDownloaded { get { return Name == DownloadedBooksCollectionNameInEnglish; } }
+
+		private string FactoryDir
```

Comment:
Is this trying to find something different from ProjectContext.FactoryCollectionsDirectory or ProjectContext.GetInstalledCollectionsDirectory()? It's confusing because I think Basic Book is normally in the former, but this code seems to be looking for it in the latter. But then, maybe I'm the one confused...these locations are different for dev and end-user machines, and I seem to have gotten it wrong in one recent PR. Still, I think we should try to have only one bit of code that knows where these directories are, and I think ProjectContext is a pretty good place for it if it isn't already there.